server/match/urls.py

```diff
@@ -20,5 +20,10 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path('users/', include('users.urls')),
     path(r"", include(talent_router.urls))
 ]
+
+urlpatterns+=[
+    path('api-auth/', include('rest_framework.urls'),),
```

Comment:
do we need this ??

Reply:
Not anymore, I guess.
I think I used it to test the fields required "username" or "email".

but i had to use `{'username': email@mail.blaah, 'password': 'abc'}` for **"/users/auth/"** (with username: email).

Reply:
so we are not using email password ??

Reply:
We are. I tried with 'email' but that didn't work until I put username.

But it takes in the email. The field name is just username.

I'll find a fix for that.

Reply:
found a fix for you, its in the comment [below](https://github.com/Taycode/match-monorepo/pull/6#discussion_r1569141136).